specifyweb/frontend/js_src/lib/utils/relativeDate.ts

```diff
@@ -1,7 +1,7 @@
 import { parseDate } from '../components/FormPlugins/PartialDateUi';
 import { mappedFind } from './utils';
 
-const reParse =
+export const reParse =
```

Comment:
Since this is now used outside of the `relativeDate`, the `reParse` name is very ambiguous.

Remember that when exporting something outside the file, you often need to make the name more verbose so that it's clear what the variable is about, and so that there aren't conflicts with other exported variables (as that confuses auto import)

Reply:
Fixed